test_ffi/tests/thread_safe_test_worker.js

```diff
@@ -35,7 +35,7 @@ self.addEventListener("message", ({ data }) => {
   } else if (data === "call") {
     dylib.symbols.call_stored_function();
   } else if (data === "unref") {
-    callback.unref();
+    callback.close();
```

Comment:
Seems like an unrelated change

Reply:
This was me trying to, again, get rid of the flake in FFI tests.